src/main/webapp/js/auscope/Main-UI.js

```diff
@@ -404,9 +418,196 @@ Ext.application({
                 popup.show();
             });
         };
-        Ext.get('permalink').on('click', permalinkHandler);
-        Ext.get('permalinkicon').on('click', permalinkHandler);
+        Ext.get('permanent-link').on('click', permalinkHandler);
+
+        //Create our advanced search control handler
+        var advancedSearchLinkHandler = function() {
```

Comment:
Could you move this out into another file.  MainUI.js should just define things (IMO) and the impl should be elsewhere.  With ExtJs I have had problems achieving this though (FOAL).